src/App.js

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import Store from './redux/Storage';
-import BookList from './Components/BookList';
-import Navigation from './Components/Navigation';
-import Categories from './Components/categories';
+import Book from 'Components/Book';
+import Categories from 'Components/Categories';
```

Comment:
- [ ] You have an import error. Kindly fix that.

  ![image](https://github.com/tchasinga/bookstoring/assets/81468675/4df1a567-0315-4e81-8242-8d1c4bc41b55)

  Should be this:

  ```
  import Categories from 'Components/categories';
  ```